csfieldguide/static/interactives/binary-cards/css/binary-cards.css

```diff
@@ -67,3 +67,6 @@
 h3, h5, p {
   text-align: left;
 }
+.binary-card {
```

Comment:
There is already a rule for this (line 16), it would be nicer to have the rule in there.